### DISCUSSION TITLE
Python 3.10 support

### DISCUSSION THREAD

**Discussion by sdebruyn** [great-expectations/great_expectations#5911] [General]
Python 3.10 has been out for about a year now but [according to the docs](https://docs.greatexpectations.io/docs/guides/setup/installation/local/) this is not supported. What would have to be done to support Python 3.10?

**Comment by austiezr** [comment]
Hey @sdebruyn ! Thanks for reaching out. We do have eyes on official support for `3.10`, and many members of the community are successfully using `3.10`. If this is something you'd be interested in contributing towards, we would love to have an ongoing discussion about next steps with the community; to that end, I'm converting this issue into a [Discussion](https://github.com/great-expectations/great_expectations/discussions).
